modules.d/98usrmount/mount-usr.sh

```diff
@@ -6,12 +6,13 @@ type fsck_single > /dev/null 2>&1 || . /lib/fs-lib.sh
 filtersubvol() {
     local _oldifs
     _oldifs="$IFS"
-    IFS=","
-    set $*
+    local IFS=","
```

Comment:
Everywhere else, you separated the  local declaration from the assignment. Why isn't shellcheck complaining here?

Reply:
It's only complaining, if a command is executed in the assignment.